Wikipedia/Code/MWKImage.m

```diff
@@ -156,10 +156,6 @@ - (NSString *)fileNameNoSizePrefix {
 
 #pragma mark - Import / Export
 
-- (void)save {
```

Comment:
Just curious, how is this handled now? I couldn't find where images are saved now in the diff

Reply:
MWKImages I should say…